frontends/ol-components/src/components/Button/Button.tsx

```diff
@@ -357,13 +362,15 @@ const actionStyles = (size: ButtonSize) => {
  */
 const ActionButton = styled(
   React.forwardRef<HTMLButtonElement, ActionButtonProps>((props, ref) => (
-    <ButtonStyled ref={ref} type="button" {...props} />
+    <button ref={ref} type="button" {...props} />
   )),
-)(({ theme, size = defaultProps.size, responsive }) => {
+)((props) => {
+  const { size, responsive } = { ...props, ...defaultProps }
   return [
+    buttonStyles(props),
```

Comment:
Previous, `ActionButton` was:
```tsx
const ActionButton = styled(
  React.forwardRef<HTMLButtonElement, ActionButtonProps>((props, ref) => (
    <ButtonStyled ref={ref} type="button" {...props} />
  ))(...)

// and, essentially
const ActionButtonLink = ActionButton.withComponent(Link)
```

This caused ActionButtonLink to be missing the styles from ButtonStyled.

Moving the button styles to its own helper fixed that. Story added.